common/tier4_state_rviz_plugin/src/autoware_state_panel.cpp

```diff
@@ -287,8 +287,10 @@ void AutowareStatePanel::onClickVelocityLimit()
 
 void AutowareStatePanel::onClickAutowareEngage()
 {
-  auto req = std::make_shared<tier4_external_api_msgs::srv::Engage::Request>();
-  req->engage = current_engage_ ? false : true;
+  using Engage = tier4_external_api_msgs::srv::Engage;
```

Comment:
I believe you don't need the left side in this case.

```suggestion
  using tier4_external_api_msgs::srv::Engage;
```